ern/types.graphql

```diff
@@ -0,0 +1,133 @@
+# This file is part of the go-meta library.
+#
+# Copyright (C) 2017 JAAK MUSIC LTD
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# If you have any questions please contact yo@jaak.io
+
+# Party Schemas
+interface PartyId {
```

Comment:
Is this used anywhere?

Reply:
Not exactly, forgot to remove, apologies. It details the fields we wish to cover. I will remove and then share with the team.